wishlist/wishlist.css

```diff
@@ -0,0 +1,186 @@
+
+.page-container{
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+}
+.navbar{
+    margin-bottom: 5px;
+    padding: 1.3rem var(--spacing3) ;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-bottom: 6px;
+    border-bottom: 1px solid rgb(207, 207, 207);
+}
```

Comment:
Suggestion to add one empty line at the end of the code block, for readability.

Reply:
Sure